galite-core/src/main/kotlin/org/kopi/galite/visual/UItemTree.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2013-2020 kopiLeft Services SARL, Tunis TN
+ * Copyright (c) 1990-2020 kopiRight Managed Solutions GmbH, Wien AT
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package org.kopi.galite.visual
+
+import org.kopi.galite.base.UComponent
+
+/**
+ * `UItemTree` is the top-level interface that represents a tree.
+ * The visible component is built from a [VTreeWindow] model.
+ */
+interface UItemTree : UWindow {
+  /**
+   * Returns The Tree instance
+   *
+   * @return a [UTreeComponent]
+   */
+  val tree: UTreeComponent
```

Comment:
i think that it should be a function to implement it later :   public UTreeComponent getTree();

Reply:
Changed to getTree() : UTreeComponent